homeassistant/components/zha/core/device.py

```diff
@@ -98,8 +99,9 @@ def __init__(
             self._zigpy_device.__class__.__module__,
             self._zigpy_device.__class__.__name__,
         )
+        keep_alive_interval = random.randint(*_UPDATE_ALIVE_INTERVAL)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5141152)